chat-widget/src/common/utils.ts

```diff
@@ -438,4 +438,8 @@ export const  formatTemplateString = (templateMessage : string, values : any) =>
     return templateMessage.replace(/{(\d+)}/g, (match, index) => {
         return typeof values[index] !== "undefined" ? values[index] : match;
     });
+};
+
+export const parseLowerCaseString = (property: string | boolean | undefined): string => {
```

Comment:
add tests to https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/common/utils.test.ts